lib/hyrax/default_admin_set/railtie.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails/railtie'
+
+module Hyrax
+  module DefaultAdminSet
+    # Connect into the boot sequence of a Rails application
+    class Railtie < Rails::Railtie
+      config.to_prepare do
+        begin
+          AdminSet.find(Hyrax::DefaultAdminSetActor::DEFAULT_ID)
```

Comment:
I have thoughts about how we might streamline this Railtie and DRY out some other code (the DefaultAdminSetActor). Mind if I toss a commit at this branch?